sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py

```diff
@@ -120,11 +121,13 @@ async def create_certificate(
         get_certificate_command = partial(self.get_certificate, certificate_name=certificate_name, **kwargs)
 
         create_certificate_polling = CreateCertificatePollerAsync(
-            get_certificate_command=get_certificate_command, interval=polling_interval
+            pipeline_response=pipeline_response,
+            get_certificate_command=get_certificate_command,
+            interval=polling_interval,
         )
         def no_op(*_, **__) -> Any:  # The deserialization callback is ignored based on polling implementation
             pass
-        return await async_poller(command, create_certificate_operation, no_op, create_certificate_polling)
+        return await AsyncLROPoller(command, create_certificate_operation, no_op, create_certificate_polling)
```

Comment:
I don't think we need an await here where we return the poller object

Reply:
This API's a little odd in that we don't actually return the poller; we return the result of awaiting `.result()` directly (awaiting the poller is just a shorthand): https://github.com/Azure/azure-sdk-for-python/blob/9cb79477740fed1d5057fe6a14cdc0ab10e332d3/sdk/core/azure-core/azure/core/polling/_async_poller.py#L182-L183
This change was just moving away from the deprecated `async_poller` function 🙂

Reply:
Ah I see! Ignore me!